src/test/kotlin/com/atlassian/performance/tools/hardware/HardwareRecommendationEngine.kt

```diff
@@ -167,7 +167,7 @@ class HardwareRecommendationEngine(
         pastFailures = BugAwareTolerance(logger),
         metric = HardwareMetric(
             scale = scale,
-            presenceJudge = VirtualUsersPresenceJudge(Ratio(0.90))
+            presenceJudge = VirtualUsersPresenceJudge(Ratio(0.70))
```

Comment:
Oof, that's worrying. This affects both the CI and the full HWRs. How many VUs did we lose? How often?

Reply:
Ah yes, apologies I hadn't noticed it was in the underlying code I'd changed not the IT itself.

I should look at making it configurable from the test.

The failure was here: https://circleci.com/gh/atlassian/jira-hardware-exploration/225

Only 79.86111111111111 % VUs were encountered